iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Boundary.java

```diff
@@ -32,32 +32,21 @@ public interface Boundary {
     double getQ();
 
     /**
-     * A Boundary could be associated with one side of a branch to determine P and Q.
-     * Get the branch side the boundary refers to.
+     * Get the equipment the boundary is associated to.
      */
-    default Branch.Side getSide() {
+    default Connectable getConnectable() {
```

Comment:
@Miora This part is confusing. We have now `getConnectable` and `getDanglingLine` that have the same behaviour. I think that we have to remove `getConnectable`.

Reply:
Agree on that. Now boundaries from the point of view of a network will always have a dangling line, removing the `getConnectable` will clarify this. Furthermore, we could simplify some points where the result was analyzed to check if the connectable was a dangling line or a tie line.

Reply:
> Agree on that. Now boundaries from the point of view of a network will always have a dangling line, removing the `getConnectable` will clarify this. Furthermore, we could simplify some points where the result was analyzed to check if the connectable was a dangling line or a tie line.

Tie line is no more a connectable in that PR.